packages/web-components/src/spinner/spinner.stories.ts

```diff
@@ -1,84 +1,97 @@
+import { Spinner as FluentSpinner } from './spinner.js';
 
-type SpinnerStoryArgs = Args;
-type SpinnerStoryMeta = Meta<SpinnerStoryArgs>;
+type Story = StoryObj<FluentSpinner>;
 
-const storyTemplate = html<SpinnerStoryArgs>`
-  <fluent-spinner appearance=${x => x.appearance} size=${x => x.size}></fluent-spinner>
+const storyTemplate = html<StoryArgs<FluentSpinner>>`
+  <fluent-spinner appearance="${story => story.appearance}" size="${story => story.size}"></fluent-spinner>
 `;
 
 export default {
   title: 'Components/Spinner',
+  render: renderComponent(storyTemplate),
   argTypes: {
     appearance: {
+      control: 'select',
       description: 'The appearance of the spinner',
+      options: ['', ...Object.values(SpinnerAppearance)],
+      mapping: { '': null, ...SpinnerAppearance },
       table: {
-        defaultValue: { summary: 'primary' },
+        category: 'attributes',
+        type: { summary: Object.values(SpinnerAppearance).join('|') },
       },
-      control: {
-        type: 'select',
-        options: Object.values(SpinnerAppearance),
-      },
-      defaultValue: 'primary',
     },
     size: {
+      control: 'select',
       description: 'The size of the spinner',
+      options: ['', ...Object.values(SpinnerSize)],
+      mapping: { '': null, ...SpinnerSize },
       table: {
-        defaultValue: { summary: 'medium' },
-      },
-      control: {
-        type: 'select',
-        options: Object.values(SpinnerSize),
+        category: 'attributes',
+        type: { summary: Object.values(SpinnerSize).join('|') },
       },
-      defaultValue: 'medium',
-    },
-  },
-  parameters: {
-    status: {
-      type: 'experimental',
     },
   },
-} as SpinnerStoryMeta;
-
-export const Spinner = renderComponent(storyTemplate).bind({});
+} as Meta<FluentSpinner>;
 
-//
-// Attribute stories
-//
+export const Default: Story = {};
 
-export const Appearance = renderComponent(html<SpinnerStoryArgs>`
-  <div>
-    <div style="padding: 20px; display: flex; align-items: center; gap: 20px;">
-      <code>primary</code>
-      <fluent-spinner appearance="primary"></fluent-spinner>
-    </div>
+export const InvertedAppearance: Story = {
+  render: renderComponent(html<StoryArgs<FluentSpinner>>`
     <div
-      style="padding: 20px; background-color: var(--colorNeutralBackgroundInverted); color: var(--colorNeutralForegroundInverted); display: flex; align-items: center; gap: 20px;"
+      style="padding: 40px; background-color: var(--colorNeutralBackgroundInverted); color: var(--colorNeutralForegroundInverted);"
     >
-      <code>inverted</code>
-      <fluent-spinner appearance="inverted" size="medium"></fluent-spinner>
+      ${storyTemplate}
     </div>
-  </div>
-`);
+  `),
+  parameters: {
+    layout: 'fullscreen',
+  },
+  args: {
+    appearance: SpinnerAppearance.inverted,
+  },
+};
+
+export const TinySize: Story = {
+  args: {
+    size: SpinnerSize.tiny,
+  },
+};
```

Comment:
I feel like to be consistent we'd put all the size options together into one story like it was previously rather than have individual stories for each option value.

Reply:
You mean having them all next to each other? This would mean we would not use the args approach and just force render 3 spinners next to one another